packages/graphql-dynamodb-transformer/src/resources.ts

```diff
@@ -145,7 +157,15 @@ export class ResourceFactory {
      */
     public makeIAMRole(tableId: string) {
         return new IAM.Role({
-            RoleName: Fn.Join('-', [tableId, 'role', Fn.GetAtt(ResourceConstants.RESOURCES.GraphQLAPILogicalID, 'ApiId')]),
+            RoleName: Fn.If(
+                ResourceConstants.CONDITIONS.HasEnvironmentParameter,
+                Fn.Join('-', [
```

Comment:
Doesn't roles have a limitation of 64 chars as well? This name seems to be fragile.

Reply:
I have limited the length of the table id.